test/jdk/java/lang/ClassLoader/loadLibraryDeadlock/TestLoadLibraryDeadlock.java

```diff
@@ -169,15 +169,19 @@ public static void main(String[] args) throws Throwable {
                 "Unable to load native library.");
 
         Asserts.assertTrue(
-                countLines(outputAnalyzer, "Class1 loaded from " + bJar) > 0,
-                "Unable to load b.jar.");
+                countLines(outputAnalyzer, "Class1 loaded from " + toLocationString(bJar)) > 0,
+                "Unable to load " + toLocationString(bJar));
 
         Asserts.assertTrue(
-                countLines(outputAnalyzer, "Class2 loaded from " + cJar) > 0,
-                "Unable to load signed c.jar.");
+                countLines(outputAnalyzer, "Class2 loaded from " + toLocationString(cJar)) > 0,
+                "Unable to load signed " + toLocationString(cJar));
 
         Asserts.assertTrue(
                 countLines(outputAnalyzer, "Signed jar loaded from native library.") > 0,
                 "Unable to load signed jar from native library.");
     }
+
+    private static String toLocationString(Path path) {
+        return path.toUri().getPath();
```

Comment:
Hello Mandy, this looks OK to me. Because the `LoadLibraryDeadlock.getLocation()` uses `URL.getPath()` to print the message in the logs, maybe for consistency and to rule out any unexpected differences, perhaps use `path.toUri().toURL().getPath()` instead?